x-pack/test/apm_api_integration/tests/settings/anomaly_detection/write_user.spec.ts

```diff
@@ -30,17 +30,14 @@ export default function apiTest({ getService }: FtrProviderContext) {
   }
 
   function deleteJobs(jobIds: string[]) {
-    return legacyWriteUserClient
-      .post(`/api/ml/jobs/delete_jobs`)
-      .send({ jobIds })
-      .set('kbn-xsrf', 'foo');
+    return ml.deleteMlJobs(jobIds);
```

Comment:
we already have a similar method
```suggestion
    return ml.deleteAnomalyDetectionJobES(jobIds);
```